tests/attention/test_decode_prefill_lse.py

```
@@ -71,5 +75,198 @@ def test_mlc_failed_case():
+        total_pages, Hkv, page_size, D, dtype=torch.bfloat16, device=device
+    )
+    v_bf16 = torch.randn(
+        total_pages, Hkv, page_size, D, dtype=torch.bfloat16, device=device
+    )
+
+    kv_indptr = torch.arange(B + 1, dtype=torch.int32, device=device) * npps
+    kv_indices = torch.arange(total_pages, dtype=torch.int32, device=device)
+    kv_last_page_len = torch.full((B,), page_size, dtype=torch.int32, device=device)
+    qo_indptr = torch.arange(B + 1, dtype=torch.int32, device=device) * qo_len
+
+    ws = lambda: torch.zeros(128 * 1024 * 1024, dtype=torch.uint8, device=device)
```

Ruff (0.15.10)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `ws` as a `def`

(E731)